src/Validation/Validation.php

```diff
@@ -522,15 +522,47 @@ public static function localizedTime($check, $type = 'datetime', $format = null)
 
     /**
      * Boolean validation, determines if value passed is a boolean integer or true/false.
+     * The list of what is considered to be boolish values, may be set via $booleanValues.
      *
      * @param string $check a valid boolean
+     * @param string $booleanValues list of valid boolish values, defaults to `[true, false, 0, 1, '0', '1']`.
      * @return bool Success
      */
-    public static function boolean($check)
+    public static function boolean($check, array $booleanValues = [])
     {
-        $booleanList = [0, 1, '0', '1', true, false];
+        $booleanValues = array_merge([true, false, 0, 1, '0', '1'], $booleanValues);
```

Comment:
A merge means the user can never remove values from the list, only add to it. How would a developer replace the list of accepted boolean values?